src/service/apiClient.js

```diff
@@ -36,6 +36,11 @@ async function createProfile(
   })
 }
 
+async function getCohorts() {
+  const res = await get('cohorts')
```

Comment:
this is a sync api call, what happens if it fails?

Reply:
This comment from Luca hasn't been addressed